sidebars.js

```diff
@@ -114,7 +114,12 @@ const sidebars = {
       type: 'ref',
       id: 'unreal/unreal_sdk',
       label: 'Unreal Engine',
-    },   
+    },
+    {
```

Comment:
Change this to `type: 'doc'` and just link to the doc directly, that way it'll stay on the main sidebar.

Then you can remove `solanaKmpSidebar.js`